docs/sources/mimir/query/query-best-practices.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Learn best practices for querying Grafana Mimir.
+title: Best practices for querying Grafana Mimir
+menuTitle: Query best practices
+weight: 1
+keywords:
+  - query
+  - best practices
+  - performance
+---
+
+<!-- Note: This topic is mounted in the GEM documentation. Ensure that all updates are also applicable to GEM. -->
+
+# Best practices for querying Grafana Mimir
+
+The way you write queries for Grafana Mimir affects the speed and quality of your results. Follow these best practices to ensure optimal performance and reliability for queries.
+
+## Use precise label selectors
+
+To reduce the initial dataset size, start with the most specific label selectors. For example, if you have labels like `namespace` and `app_name` where `app_name` is more specific, lead with `app_name="myapp"` rather than starting with the broader `namespace` selector.
+
+To further reduce the number of series that require processing, include as many label selectors as possible for your query.
```

Comment:
This is true up to a point: if extra selectors don't help filter the amount of data, then they can be counter-productive.

eg. if `metric{cluster="prod", region="us"}` returns the same results as `metric{cluster="prod", region="us", provider="aws"}`, then the form with fewer selectors is slightly more efficient.